ui-manchette/src/components/OperationalPoint.tsx

```diff
@@ -27,7 +32,7 @@ const OperationalPoint = ({
       id={id}
       ref={opRef}
       onClick={() => {
-        if (onClick) onClick(id, opRef.current);
+        if (onClick) onClick(`${id}-${position}`); // to handle waypoints with same id
```

Comment:
Hm, passing in here a custom string which isn't an ID seems pretty surprising… Especially since the argument is named `opId` in the type definition.

But why do we need to pass any argument here? The callers already provide a callback per-OP, no?

Reply:
Hmm but if we don't pass any arguments, how can we have the information of which waypoint has been clicked ?

Reply:
Ah, apologies, I mixed things up with the menu item callback!